internal/server/keyRingEndpoint.go

```diff
@@ -75,6 +84,14 @@ func (kre *KeyRingEndpoint) getKey(w http.ResponseWriter, r *http.Request) {
 	LogExit(log, r, 200, n)
 }
 
+// Get Cached Keys godoc
+// @id kre-get-cached-keys
+// @tags Key Ring
+// @Summary Fetches a list of cached keys in the server
```

Comment:
```Fetches a list of cached keys from the server```
Mesmo comentário sobre o server? é um Keyserver específico?
Se for uma variável de ambiente, acho que poderia até colocar o nome da variável.